less/about/preferences.less

```diff
@@ -455,6 +455,46 @@ table.sortableTable {
   overflow-y: scroll;
 }
 
+.modal .dialog.paymentHistory .sectionTitle {
+  text-align: left;
+}
+
+span.paymentHistoryButton.browserButton {
```

Comment:
@willy-b before the wallet is created, we have no idea what the user's currency is, so it defaults to BTC. it could just say 'empty' or something.